src/components/NurseryWithdrawals/PlantingProgressList.tsx

```diff
@@ -71,31 +76,91 @@ const columnsWithZones = (): TableColumnType[] => [
 export type PlantingProgressListProps = {
   search: string;
   plantingCompleted?: boolean;
+  reloadTracking: () => void;
 };
 
-export default function PlantingProgressList({ search, plantingCompleted }: PlantingProgressListProps): JSX.Element {
+export default function PlantingProgressList({
+  search,
+  plantingCompleted,
+  reloadTracking,
+}: PlantingProgressListProps): JSX.Element {
   const [hasZones, setHasZones] = useState<boolean | undefined>();
   const classes = useStyles();
   const data = useAppSelector((state: any) => searchPlantingProgress(state, search.trim(), plantingCompleted));
+  const [selectedRows, setSelectedRows] = useState<any[]>([]);
+  const dispatch = useAppDispatch();
+  const [requestId, setRequestId] = useState<string>('');
+  const selector = useAppSelector((state) => selectUpdatePlantingsCompleted(state, requestId));
 
   useEffect(() => {
     if (data && hasZones === undefined) {
       setHasZones(data.some((d) => d.subzoneName));
     }
   }, [data, hasZones]);
 
+  useEffect(() => {
+    if (selector) {
+      if (selector.status === 'success') {
+        reloadTracking();
+      }
+    }
+  }, [selector, reloadTracking]);
+
   if (!data || hasZones === undefined) {
     return <CircularProgress sx={{ margin: 'auto' }} />;
   }
 
+  const undoPlantingComplete = () => {
+    const subzoneIds = selectedRows.map((row) => row.subzoneId);
+    const request = dispatch(requestUpdatePlantingsCompleted({ subzoneIds, planting: { plantingCompleted: false } }));
+    setRequestId(request.requestId);
+  };
+
+  const setPlantingComplete = () => {
+    const subzoneIds = selectedRows.map((row) => row.subzoneId);
+    const request = dispatch(requestUpdatePlantingsCompleted({ subzoneIds, planting: { plantingCompleted: true } }));
+    setRequestId(request.requestId);
+  };
+
+  const getTopBarButtons = () => {
+    const topBarButtons: TopBarButton[] = [];
+
+    if (selectedRows.length) {
+      const areAllIncompleted = selectedRows.every((row) => row.plantingCompleted === false);
+      const areAllCompleted = selectedRows.every((row) => row.plantingCompleted === true);
+
+      topBarButtons.push({
+        buttonType: 'passive',
+        buttonText: strings.UNDO_PLANTING_COMPLETE,
+        onButtonClick: () => undoPlantingComplete(),
```

Comment:
one suggestion, here and the undo call.. this happens so fast user has no idea if it even completed. We should do something similar to accession check-in ux. When it is over we should show a status modal. Feel free to check with @chudo76
Here's the accession check-in modal confirmation:

<img width="308" alt="Terraware App 2023-07-19 13-24-45" src="https://github.com/terraware/terraware-web/assets/1865174/f57ff7b6-f59d-4472-b063-501879e5c320">

Reply:
cc @constanzauanini